packages/graphiql-explorer/src/index.ts

```diff
@@ -0,0 +1,6 @@
+export type { Props as GraphiQLExplorerProps } from './explorer';
+export {
+  Explorer as ExplorerInner,
+  ExplorerWrapper as default,
+  ExplorerWrapper as Explorer,
+} from './explorer';
```

Comment:
@sgrove keeping the exports in parity here, but also exporting the inner component as we have our own plugin error boundary planned